sns_ik_examples/src/ik_tests.cpp

```diff
@@ -217,15 +220,17 @@ void test(ros::NodeHandle& nh, double num_samples, std::string chain_start, std:
   std::vector<KDL::JntArrayVel> JointVelList;
   KDL::JntArrayVel v(chain.getNrOfJoints());
   // Create Random velocities within the vel limits
-  for (uint i=0; i < num_samples; i++) {
+  for (uint i=0; i < num_samples_vel; i++) {
```

Comment:
It did originally seg fault, but I put a check for that in line 112 tested it both ways.